static/docs/tutorials/index.md

```diff
@@ -0,0 +1,74 @@
+# Tutorials
+
+## Basic Concepts
+
+Learn basic concepts and features of DVC with interactive lessons:
```

Comment:
as I mentioned let's do this as a single entry within Interactive section. We put too much details for the index page. Especially considering that we have a full section basics.md with the same content.